File: UnicoVehicle/UnicoVehicle/UnicoVehicle.DTO/VehicleStock.cs
using System;
namespace UnicoVehicle.DTO
{
    public class VehicleStock
    {
        public int VehicleStockId { get; set; }
        public Miscellaneous.Vehicle Vehicle { get; set; }
        public Miscellaneous.Showroom Showroom { get; set; }
        public int Stock { get; set; }
    }
}

File: Flutter App/unicoVehicle/lib/Models/VehicleStock.dart
import './Miscellaneous/Showroom.dart';
import './Miscellaneous/Vehicle.dart';

class VehicleStock {
  final int? vehicleStockId;
  final Showroom? showroom;
  final Vehicle? vehicle;
  final int? stock;

  VehicleStock({
    this.showroom,
    this.stock,
    this.vehicle,
    this.vehicleStockId,
  });
}
